hybrid/ddc_bounds.c

```diff
@@ -0,0 +1,21 @@
+#include "../include/common.h"
```

Comment:
I *think* this file (which is missing a comment?) should be called `ddc_out_of_bounds`?

Reply:
Sure. It does not have a comment yet because I wanted to address @jacobbramley's comment first. It works but I am not sure if it is the intended behaviour.